app.js

```diff
@@ -26,6 +26,14 @@ if (!process.env.RACKSPACE_USERNAME
   throw new Error('Required parameters not provided from the environment');
 }
 
+routes.client.createContainer({
```

Comment:
Initially I was going to do that, but if the container already exists, then a `create` operation returns a 202 and does nothing. My reasoning is that there has to be an HTTP call, so we might as well use 1 instead of 2.